python/dolfinx/fem/function.py

```diff
@@ -485,7 +480,7 @@ def copy(self) -> Function:
     @property
     def x(self) -> la.Vector:
         """Vector holding the degrees-of-freedom."""
-        return la.Vector(self._cpp_object.x)  # type: ignore
+        return self._x  # type: ignore
 
     @property
     def vector(self):
```

Comment:
We should probably deprecate this method.

Reply:
Yes, we can deprecate the method. However, I would like to get consensus on this, since it will require to update all the python examples using `Function.vector` to use `Function.x.petsc_vec`

Reply:
I think we should put in a deprecation-warning, and remove it with the release of 0.9.x

Reply:
Sounds good. I'll do that and replace all occurrences of `u.vector` to `u.x.petsc_vec`.

Reply:
@garth-wells @jorgensd : This has been addressed. All occurrences of `u.vector` were replaced with  `u.x.petsc_vec` and a deprecation warning was added to `dlx.fem.Function.vector`.